update_opcodes.py

```diff
@@ -69,6 +69,7 @@ def get_source_path(crate, name):
 opcode_dest_path = get_source_path('stencil', 'opcode.rs')
 emitter_dest_path = get_source_path('emitter', 'emitter.rs')
 function_dest_path = get_source_path('stencil', 'function.rs')
+script_dest_path = get_source_path('stencil', 'script.rs')
```

Comment:
Okay, I'll rename it to `update_stencil.py`.  given that what we should sync is stencil part.
and emitter is updated because the stencil part is changed.